doc/source/methods/CFRL.ipynb

```diff
@@ -0,0 +1,475 @@
+{
```

Comment:
<p>Add a line to import <code>get_he_preproccesor</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/457/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>